modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -92,12 +92,17 @@ class ConvolutionBackpropDataExtendedLayerTest
                                                           bool addBiases = false,
                                                           const std::vector<float> &filterWeights = {},
                                                           const std::vector<float> &biasesWeights = {}) {
-        bool randomFilterWeights = filterWeights.empty();
         auto shape = in.get_shape();
         std::vector<size_t> filterWeightsShape = {shape[1], numOutChannels};
         filterWeightsShape.insert(filterWeightsShape.end(), filterSize.begin(), filterSize.end());
-        auto filterWeightsNode =
-            ngraph::builder::makeConstant(type, filterWeightsShape, filterWeights, randomFilterWeights);
+        std::shared_ptr<ov::op::v0::Constant> filterWeightsNode;
+        if (filterWeights.empty()) {
+                ov::Tensor random_tensor(type, filterWeightsShape);
+                ov::test::utils::fill_tensor_random(random_tensor);
+                auto constant = std::make_shared<ov::op::v0::Constant>(random_tensor);
```

Comment:
Where `constant` is used? Should it be `filterWeightsNode`?

Reply:
My fault. Fixed